receiver/k8sobjectsreceiver/unstructured_to_logdata.go

```diff
@@ -37,6 +34,7 @@ func watchObjectsToLogData(event *watch.Event, observedAt time.Time, config *K8s
 		if name != "" {
 			attrs.PutStr("event.domain", "k8s")
 			attrs.PutStr("event.name", name)
+			attrs.PutStr("event.type", string(event.Type))
```

Comment:
FYI: there is an issue regarding reflecting of k8s events to the events API definition - https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/35857#issuecomment-2420214811, maybe it makes sense to include `event.type` to the events api convention